.github/workflows/check-v4.yml

```diff
@@ -0,0 +1,46 @@
+name: check v4
```

Comment:
If we can sync up the indentation w/ v3, it'd be easier to keep them in sync (w/o ignoring whitespace)

Reply:
Fixed.